src/wp-includes/comment-template.php

```diff
@@ -1927,17 +1927,21 @@ function post_reply_link( $args = array(), $post = null ) {
  *
  * @since 2.7.0
  *
- * @param string $text Optional. Text to display for cancel reply link. If empty,
- *                     defaults to 'Click here to cancel reply'. Default empty.
+ * @param string      $text Optional. Text to display for cancel reply link. If empty,
+ *                          defaults to 'Click here to cancel reply'. Default empty.
+ * @param int|WP_Post $post Optional. The post the comment thread is being
```

Comment:
Also needs to `null` for the `@param`. Why? 

- `null` is the default
- it is acceptable to pass `null`
- for parity with `get_post()` which documents this param as `int|WP_Post|null`